apps/src/sites/code.org/pages/public/yourschool.js

```diff
@@ -9,9 +9,9 @@ import responsive from '@cdo/apps/code-studio/responsiveRedux';
 
 registerReducers({isRtl, responsive});
 
-$(document).ready(showYourSchool);
+$(document).ready(initYourSchool);
 
-function showYourSchool() {
+function showYourSchool(schoolDropdownOption) {
```

Comment:
It looks like you never pass an argument in the places this function is called - is this planning for a future change?

Reply:
I'll remove this in the followup. This used to be how the JS code in the haml file passed the school selection to the `YourSchool` React component. It isn't needed anymore.